joy2twist/launch/gamepad_controller.launch.py

```diff
@@ -0,0 +1,45 @@
+import os
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, ExecuteProcess, IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+
+    joy2twist_cfg_path = os.path.join(get_package_share_directory(
```

Comment:
We agreed on not to use `os.path.join()`

Reply:
My omission - corrected :heavy_check_mark: